src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3290,7 +3300,7 @@ private void ReportNullabilityMismatchInArgument(BoundExpression argument, TypeS
                 new FormattedSymbol(parameter.ContainingSymbol, SymbolDisplayFormat.MinimallyQualifiedFormat));
         }
 
-        // PROTOTYPE(NullableReferenceTypes): If support for [NullableOptOut] or [NullableOptOutForAssembly]
+        // PROTOTYPE(NullableReferenceTypes): If support for [NonNullTypes] or [NullableOptOutForAssembly]
```

Comment:
>for [NonNullTypes] or [](start = 57, length = 21)

Does `[NonNullTypes]` affect this method?

Reply:
I removed `GetTypeOrReturnTypeWithAdjustedNullableAnnotations`. It's in a separate commit (number 8).

---
In reply to: [196956897](https://github.com/dotnet/roslyn/pull/27807#discussion_r196956897) [](ancestors = 196956897)